x-pack/plugins/observability_solution/inventory/public/components/search_bar/index.tsx

```diff
@@ -65,8 +65,11 @@ export function SearchBar() {
         entityTypes,
         refresh: !isUpdate,
```

Comment:
You can remove this `refresh` from `searchBarContentSubject$` observable as we now have a dedicated one.